app/models/user.rb

```diff
@@ -11,7 +11,7 @@ class User < ActiveRecord::Base
             presence: true
   validates :staff, inclusion: { in: [true, false],
                                  message: 'must be true or false' }
-  validates :spire, uniqueness: true
+  validates :spire, uniqueness: true, format: { with: /\A\d{8}@umass\.edu\z/ }
```

Comment:
Oh good point — would you mind opening an issue (or PR) making sure we're checking beginning and end of string on screaming-dinosaur?